Sources/DarkModeCore/DMDynamicColor.m

```diff
@@ -19,25 +18,48 @@ @implementation DMDynamicColorProxy
 
 // TODO: We need a more generic initializer.
 - (instancetype)initWithLightColor:(UIColor *)lightColor darkColor:(UIColor *)darkColor {
-  self.lightColor = lightColor;
-  self.darkColor = darkColor;
+  return [self initWithDynamicProvider:^(DMTraitCollection *traitCollection){
```

Comment:
The original light/dark color initialization is forwarded to the dynamic provider init method also for consistency